packages/uploadthing/src/internal/error-formatter.ts

```diff
@@ -1,7 +1,21 @@
 import type { UploadThingError } from "@uploadthing/shared";
 
+import type { FileRouter, inferErrorShape } from "./types";
+
 export function defaultErrorFormatter(error: UploadThingError) {
   return {
     message: error.message,
   };
 }
+
+export function formatError<TRouter extends FileRouter>(
+  error: UploadThingError,
+  router: TRouter,
+): inferErrorShape<TRouter> {
+  const errorFormatter =
+    router[Object.keys(router)[0]]?._def.errorFormatter ??
```

Comment:
hmmmm this seems a bit sketchy

Reply:
Haha right? It's the same code as before though 😅